### ISSUE TITLE
-leg pass multiple bills 

### ISSUE THREAD

**Issue by ArchWizard56** [jonasbohmann/democraciv-discord-bot#12]
It would be a useful feature for the Speaker to be able to mark multiple bills as passed with one use of `-leg pass`.

**Comment by jonasbohmann** [comment]
Agreed, probably same with `-ministry pass ` and `-ministry veto`.

**Comment by jonasbohmann** [comment]
A "proof-of-concept" implemenation of this was added in 21b975d. 

If the command argument is longer than one id, the argument is converted into a `MultipleBills` object, which contains a list of `Bill` objects. Then I just recursively invoke the same command with each bill from `MultipleBills`.

That works just fine but the bot sends out one message per command invokation, either an error message or the "Are you sure?" question, meaning the person still has to confirm every single bill.

So I'm probably not going to do it that way and instead make the bot just invoke the command once thus ask only once "Are you sure you want to pass/veto _all_ these bills?".

**Comment by jonasbohmann** [comment]
This was properly implemented in the 0.17 update now, which was merged into master today.
